docker/Dockerfile

```diff
@@ -14,7 +14,15 @@ WORKDIR ${HOME}
 
 # Install base dependencies
 RUN apt-get update && \
-    apt-get install -y curl git 
+    apt-get install -y curl git wget build-essential
+
+# Install cmake which is a requirement by xlearn 
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.15.2/cmake-3.15.2.tar.gz && \
```

Comment:
might be good to keep the url as an arg like we do in anaconda
```
ARG CMAKE="https://github.com/Kitware/CMake/releases/download/v3.15.2/cmake-3.15.2.tar.gz"
RUN wget ${CMAKE} -O cmake.tgz && \
    mkdir cmake && \
    tar xzf cmake.tgz --strip-components=1 -C cmake && \
    cd cmake && \
    ./bootstrap && \
    make && \
    make install
    cd .. && \
    rm cmake.tgz && \
    rm -rf cmake
```

Reply:
Make sense 👍